anvil/src/eth/miner.rs

```diff
@@ -81,32 +78,24 @@ impl Miner {
 #[derive(Debug)]
 pub struct MinerInner {
     waker: AtomicWaker,
-    set: AtomicBool,
 }
 
 // === impl MinerInner ===
 
 impl MinerInner {
     /// Call the waker again
     fn wake(&self) {
-        self.set.store(true, Relaxed);
         self.waker.wake();
     }
```

Comment:
that change makes perfect sense.
the `set` flag basically ensured it can be only woken up once

Reply:
Actually I think this was an implementation error, since it just ensured that the context can be set only once (it was only `set` in the `wake` method, but not checked)

Reply:
definitely, thanks for fixing